site/docs/guide/context.md

```diff
@@ -558,11 +559,7 @@ You can even mix additive and transformative flavors:
 
 ```ts
 type MyContext = FlavorX<
-  FlavorY<
-    FlavorZ<
-      Context & FlavorA & FlavorB & FlavorC
-    >
-  >
+  FlavorY<FlavorZ<Context & FlavorA & FlavorB & FlavorC>>
```

Comment:
Was it done by accident?

Reply:
The CLI applies different formatting than the LSP. I ended up accepting this now, but mostly because I didn't feel like figuring it out. If there's a misconfiguration of VSC then I'm happy to revert and keep it the old way.

Reply:
I think the problem is on your side. In any case, this change should be reverted

Reply:
This change has been reverted (https://github.com/grammyjs/website/pull/1048/commits/7109874cc82cc3594c6e2aac9f20b9b44259378d) as disccussed in Telegram https://t.me/c/1322374388/6528